src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends FigureColor {
```

Comment:
The class name 'Square' is appropriate as it is a noun, but ensure it properly implements the required interfaces and extends the correct classes according to the design.